easy_handeye/scripts/publish.py

```diff
@@ -11,8 +11,10 @@
     pass
 
 inverse = rospy.get_param('inverse')
+filename = rospy.get_param('calibration_file')
+print("filename", filename)
```

Comment:
Please remove the `print` (or rather convert it to a more verbose `rospy.loginfo`), and if the value fetched from the parameter server is empty, replace it with the previous default value to maintain backwards compatibility

Reply:
Done. Let me know if that works for you.